android/src/main/res/drawable-v26/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't see this file used anywhere, can it be removed?

Reply:
Looks like `mipmap-anydpi-v26/ic_launcher.xml` references a file with this name. But that is in the mipmap folder.

Reply:
Ah I see, then this comment can be ignored. Thanks for finding it.